src/AST-Core/ASTCacheMissStrategy.class.st

```diff
@@ -16,5 +16,5 @@ Class {
 
 { #category : #accessing }
 ASTCacheMissStrategy >> getASTFor: aCompiledMethod [
-	^ aCompiledMethod parseTree doSemanticAnalysisIn: aCompiledMethod methodClass
+	^ aCompiledMethod parseTree
```

Comment:
With little fix in #parseTree method (see below) the extra semantic analysis is not needed any more.